app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes, foreign_key: 'author_id'
```

Comment:
- [ ] Good job on including the foreign key for the`likes`. Please also add it to the `comments` and `posts` as recommended by the previous reviewer.

Reply:
Done. Thanks